scripts/env.api

```diff
@@ -0,0 +1,28 @@
+# Copy this file to .env
```

Comment:
Do we need to copy it? I saw the exactly this file name being referenced in the compose...

Reply:
Removed. I copied it from previous `.env` file.